BE/PoroDev.Runtime/Services/ZipManipulator.cs

```diff
@@ -0,0 +1,65 @@
+﻿using PoroDev.Common.Exceptions;
+using PoroDev.Runtime.Services.Contracts;
+
+namespace PoroDev.Runtime.Services
+{
+    public class ZipManipulator : RuntimePathsAbstract, IZipManipulator
+    {
+        public ZipManipulator() : base()
+        {
+
+        }
+
+        public ZippedFileException Initialize(string runtimeFolderPath)
+        {
+            return (ZippedFileException)SetFolderPath(runtimeFolderPath);
+        }
+
+        public ZippedFileException ExtractZipToPath()
+        {
+            try
+            { 
+                System.IO.Compression.ZipFile.ExtractToDirectory(ZippedFilePath, RuntimeFolderPath);
```

Comment:
Should be in using "System.IO.Compression.ZipFile"?

Reply:
done